cli/command/cluster/health.go

```diff
@@ -62,81 +65,49 @@ func runHealth(storageosCli *command.StorageOSCli, opt *healthOpt) error {
 		}
 	}
 
-	if err := cliTypes.SortCLINodes(cliTypes.ByNodeName, nodes); err != nil {
-		return err
-	}
-
-	for _, node := range nodes {
-		if err := runNodeHealth(node, opt.timeout); err != nil {
-			return err
-		}
-	}
-
 	clusterHealthCtx := formatter.Context{
 		Output: storageosCli.Out(),
 		Format: formatter.NewClusterHealthFormat(format, opt.quiet),
 	}
-	return formatter.ClusterHealthWrite(clusterHealthCtx, nodes)
-}
-
-func runNodeHealth(node *cliTypes.Node, timeout int) error {
-	addr := node.AdvertiseAddress
-
-	u, err := url.Parse(node.AdvertiseAddress)
-	if err == nil && u.Host != "" {
-		addr = u.Host
+	if err := formatter.ClusterHealthWrite(clusterHealthCtx, status); err != nil {
+		return err
+	} else if opt.errReturn {
```

Comment:
Would be cleaner to remove the `else if` and just do a new `if opt.errReturn`?

Reply:
fixed